Source/Core/Core/PowerPC/Interpreter/Interpreter_LoadStore.cpp

```diff
@@ -792,7 +788,7 @@ void Interpreter::stfiwx(UGeckoInstruction inst)
     return;
   }
 
-  PowerPC::Write_U32((u32)riPS0(inst.FS), address);
+  PowerPC::Write_U32(static_cast<u32>(rPS(inst.FS).ps0), address);
```

Comment:
maybe the u32 casts/ConvertToSingle could be made other getters?

Reply:
Sure thing